lib/rules/jest/no-vague-titles.js

```diff
@@ -84,14 +84,8 @@ function getMethodName({callee}) {
       return callee.name;
     case 'MemberExpression':
       return callee.object.name;
-    case 'ArrowFunctionExpression':
-      return '';
-    case 'Import':
-      return 'import';
-    case 'Super':
-      return 'super';
     default:
-      throw new Error('Could not get method name from node.');
+      return '';
```

Comment:
I don't know what benefit we get by throwing here. We can just check for the things we care about and return no method name for the things we don't.

Reply:
I'm guessing only once we branch into `containsVagueWord` do we absolutely need the method name.

Reply:
I think maybe return false instead here so it is significantly different than the other return values

Reply:
alternatively: return a symbol named something like `NO_METHOD_NAME`, then instead of falsiness just check against that.